core/src/saros/session/User.java

```diff
@@ -1,5 +1,7 @@
 package saros.session;
 
+import java.util.*;
```

Comment:
Our code should be formatted according to [google java style](https://google.github.io/styleguide/javaguide.html), which prohibits the use of [wildcard imports](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports).

In general, the code (including wildcard imports) should be formatted correctly by default when developing in Eclipse or IntelliJ environments that were set up according to our [guide](https://www.saros-project.org/contribute/development-environment.html). (Eclipse formats on saving, IntelliJ only formats when explicitly triggering it using the keyboard shortcut `ctrl+alt+l` (see [IntelliJ documentation](https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html)).)